Assets/HoloToolkit/Utilities/README.md

```diff
@@ -14,6 +14,16 @@ A drop in direction indicator that stays in the users view at all times.
 #### HeadsUpDirectionIndicatorPointer.prefab
 A quad based pointer to be used with the HeadsUpDirectionIndicator prefab to create an out of box direction indicator.
 
+###[Scenes](Scenes)
+---
+
+####HeadsetAdjustment.unity
```

Comment:
@NeerajW - Any way you can respond here?  I would love to see this move forward into the Toolkit...